st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -87,7 +88,7 @@ def post(self, payload):
                                              notify=notify,
                                              context=context)
 
-        return str(execution.id)
+        return execution
```

Comment:
I am happy to see this changed but what is the usecase? e.g. why do we need more than an execution-id at this point?

The need for consistency with /executions is not obvious to me - can you guys explain why it is desired?